levels/pages/faq/faq.jade

```diff
@@ -335,3 +337,48 @@ block content
                 <div class='block__elem3'></div>
             </div>
             ```
+
+        a(id="global-css-resets")
+        .raw-text
+          :markdown
+            ## I've heard that BEM does not recommend global CSS resets. Why?
+
+            > CSS resets is a practise making a good showing. Many frameworks first align anything and then
+            > apply their special styles. BEM does not recommend common resets. Why? And what we are supposed
+            > to do instead?
+
+            Nothing bad would happen to your blocks if you use common reset (well, except of some special cases below).
+            So, BEM does not prohibit to use them. But using them BEM-way would be more effective.
+
+            Common CSS reset is a set of CSS to be applied to document nodes and ensure that their default view is the
+            same in different browsers. In most cases the CSS rules are written for tag selectors and this is not recommended
+            in BEM (you can find a lot of explanation above).
+
+            Another point is that in BEM a block encapsulates everything which is needed for it to be displayed and function.
+            And this is why we call the BEM blocks independent. If the block does not look properly without a third-party
+            CSS being added onto the page, it cannot be called "independent§" that much.
+
+            Assuming this all, BEM recommends every block to reset itself. If you have `menu` block and `list` block both
```

Comment:
I'm trying hard to comprehend this - `BEM recommends every block to reset itself`, but without success. Why block independence forces us to include resets in every block? Why not include local values to properties, that need it? Like this:

```css
.menu {
    padding: 0;
    margin: 0;
}
```